sycl/source/detail/graph/graph_impl.hpp

```diff
@@ -161,7 +161,7 @@ class graph_impl : public std::enable_shared_from_this<graph_impl> {
   /// Create an empty node in the graph.
```

Comment:
Couldn't update the one above because it copies `shared_ptr`s into the `Handler.impl->MNodeDeps`.

Reply:
Think that's an interesting enough note to put in the doxy comment.